WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -1101,8 +1100,27 @@ private extension ProductFormViewController {
         guard let site = ServiceLocator.stores.sessionManager.defaultSite else {
             return
         }
-        let viewModel = BlazeWebViewModel(source: .productMoreMenu, siteURL: site.url, productID: product.productID)
-        let webViewController = AuthenticatedWebViewController(viewModel: viewModel)
+
+        if viewModel.shouldShowBlazeIntroView {
+            let blazeCampaignIntroView = BlazeCampaignIntroView(onStartCampaign: { [weak self] in
+                guard let self else { return }
+                self.dismiss(animated: true)
+                self.navigateToBlazeCampaignCreation(siteUrl: site.url)
+            }, onDismiss: { [weak self] in
+                guard let self = self else { return }
+                self.dismiss(animated: true)
+            })
+
+            let hostingController = UIHostingController(rootView: blazeCampaignIntroView)
+            present(hostingController, animated: true)
```

Comment:
We should track `blazeEntryPointDisplayed` with source `introView` here.

Reply:
Added in https://github.com/woocommerce/woocommerce-ios/pull/11011/commits/b7549657f8e735762c8985a38a5655f4ac433f0a